### ISSUE TITLE
Is there a Russian language

### ISSUE THREAD

**Issue by YuraMukhin2808** [TeamHypersomnia/Hypersomnia#273]

**Comment by geneotech** [comment]
Hey there! Right now only English language is available. 
However, the game can render Cyrillic and you can use it when choosing nicknames, server names and during chat.

**Comment by GermanAizek** [comment]
> Hey there! Right now only English language is available. However, the game can render Cyrillic and you can use it when choosing nicknames, server names and during chat.

multilingual .po files would help with UI translation on any language.

**Comment by geneotech** [comment]
> multilingual .po files would help with UI translation on any language.

Absolutely, there's certainly a flexible way to do this - the real bottleneck is to find and replace all instances of hard-coded English text in code, which right now would take a massive amount of work and testing.

**Comment by shinebrillant** [comment]
Hello, I hope to add more language options to attract more people to pay attention. I can help add Traditional Chinese translation. I just hope that my friends and I can lower the threshold when playing.

**Comment by geneotech** [comment]
> Hello, I hope to add more language options to attract more people to pay attention. I can help add Traditional Chinese translation. I just hope that my friends and I can lower the threshold when playing.

Hello, this makes me really happy! Unfortunately there's still no flexible way to do this, I would have to work on the game so it's able to switch between languages, even though there are easy technologies to do this, it's just the game has a bit of tech debt in that regard.
